.github/workflows/markdown_to_pdf.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs to PDF
+on:
+  push:
+    branches:
+      - master
+    # Paths can be used to only trigger actions when you have edited certain files, such as a file within the /docs directory
+    paths:
+      - 'src/**.md'
+      - 'src/**'
+      
+  workflow_dispatch:
+    
+jobs:
+  converttopdf:
+    name: Build PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: baileyjm02/markdown-to-pdf@v1
+        with:
+          input_dir: src
+          output_dir: pdfs
+          images_dir: src/
+          # for example <img src="./images/file-name.png">
+          image_import: ./
+          # Default is true, can set to false to only get PDF files
+          build_html: false
+      - uses: actions/upload-artifact@v1
+        with:
+          name: docs
+          path: pdfs
```

Comment:
I got it now, `name` is only an identifier, but `path` has to be the full path of the final file, see <https://github.com/actions/upload-artifact#upload-an-individual-file>. As such, we would like to have something like

```
- uses: actions/upload-artifact@v2
  with:
    name: overreact-guide-pdf
    path: overreact-guide.pdf
```

(Observe the above is `upload-artifact@v2`.)

Reply:
You're right, I'll check out and change it, @schneiderfelipe

Reply:
@schneiderfelipe Solved by 244fb29be606bf71af8f7a149aaa7bc2a30fec00